{"uuid":"855869df-ba41-4bf5-ac37-161c35dadb58","historyId":"e4beccd917974399b5975fd37a907765","fullName":"org.tests.HumidityAndRainTests.setup","labels":[{"name":"package","value":"org.tests.HumidityAndRainTests"},{"name":"testClass","value":"org.tests.HumidityAndRainTests"},{"name":"testMethod","value":"setup"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"/home/kareem/IdeaProjects/Technical Test - Section2"},{"name":"subSuite","value":"org.tests.HumidityAndRainTests"},{"name":"host","value":"Kareem"},{"name":"thread","value":"66776@Kareem.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"epic","value":"Regression Tests"},{"name":"feature","value":"Humidity and Rain tests"},{"name":"AS_ID","value":"-1"}],"links":[],"name":"setup","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The application at 'src/main/resources/Android.SauceLabs.Mobile.Sample.app.2.7.1.apk' does not exist or is not accessible \nHost info: host: 'Kareem', ip: '127.0.1.1'\nBuild info: version: '4.27.0', revision: 'd6e718d134'\nSystem info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.8.0-51-generic', java.version: '21.0.1'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:app: src/main/resources/Android...., appium:appActivity: com.tohsoft.weather.ui.main..., appium:appPackage: com.wftab.weather.forecast, appium:automationName: UIAutomator2, appium:deviceName: KareemPhone, appium:noReset: true, platformName: ANDROID}]}]\nCapabilities {appium:app: src/main/resources/Android...., appium:appActivity: com.tohsoft.weather.ui.main..., appium:appPackage: com.wftab.weather.forecast, appium:automationName: UIAutomator2, appium:deviceName: KareemPhone, appium:noReset: true, platformName: ANDROID}","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The application at 'src/main/resources/Android.SauceLabs.Mobile.Sample.app.2.7.1.apk' does not exist or is not accessible \nHost info: host: 'Kareem', ip: '127.0.1.1'\nBuild info: version: '4.27.0', revision: 'd6e718d134'\nSystem info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.8.0-51-generic', java.version: '21.0.1'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:app: src/main/resources/Android...., appium:appActivity: com.tohsoft.weather.ui.main..., appium:appPackage: com.wftab.weather.forecast, appium:automationName: UIAutomator2, appium:deviceName: KareemPhone, appium:noReset: true, platformName: ANDROID}]}]\nCapabilities {appium:app: src/main/resources/Android...., appium:appActivity: com.tohsoft.weather.ui.main..., appium:appPackage: com.wftab.weather.forecast, appium:automationName: UIAutomator2, appium:deviceName: KareemPhone, appium:noReset: true, platformName: ANDROID}\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:114)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)\n\tat io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:176)\n\tat io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:237)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)\n\tat io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:270)\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:91)\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:103)\n\tat io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)\n\tat org.utils.AppiumDriverBuilder.getAppiumDriver(AppiumDriverBuilder.java:23)\n\tat org.tests.HumidityAndRainTests.setup(HumidityAndRainTests.java:35)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1737068095409,"stop":1737068095410}